x-pack/solutions/security/plugins/security_solution/public/onboarding/components/onboarding_body/cards/rules/rules_card.tsx

```diff
@@ -5,23 +5,41 @@
  * 2.0.
  */
 
-import React, { useCallback, useMemo } from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
 import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiLink, EuiSpacer } from '@elastic/eui';
 import { SecurityPageName } from '@kbn/security-solution-navigation';
 import { SecuritySolutionLinkButton } from '../../../../../common/components/links';
 import { OnboardingCardId } from '../../../../constants';
 import type { OnboardingCardComponent } from '../../../../types';
-import { OnboardingCardContentImagePanel } from '../common/card_content_image_panel';
+import { OnboardingCardContentAssetPanel } from '../common/card_content_asset_panel';
 import { CardCallOut } from '../common/card_callout';
+
 import { CardSubduedText } from '../common/card_subdued_text';
-import rulesImageSrc from './images/rules.png';
 import * as i18n from './translations';
+import type { CardSelectorListItem } from '../common/card_selector_list';
+import { CardSelectorList } from '../common/card_selector_list';
+import { useOnboardingContext } from '../../../onboarding_context';
+import { RULES_CARD_ITEMS_BY_ID, RULES_CARD_ITEMS } from './rules_card_config';
+import { DEFAULT_RULES_CARD_ITEM_SELECTED } from './constants';
+import type { CardSelectorAssetListItem } from '../types';
+import { useStoredSelectedCardItemId } from '../../../hooks/use_stored_state';
 
 export const RulesCard: OnboardingCardComponent = ({
   isCardComplete,
   setExpandedCardId,
   isCardAvailable,
 }) => {
+  const { spaceId } = useOnboardingContext();
+
+  const [toggleIdSelected, setStoredSelectedRulesCardItemId] = useStoredSelectedCardItemId(
+    'rules',
+    spaceId,
+    DEFAULT_RULES_CARD_ITEM_SELECTED.id
+  );
+  const [selectedCardItem, setSelectedCardItem] = useState<CardSelectorAssetListItem>(
```

Comment:
And this one as well, same as https://github.com/elastic/kibana/pull/199311/files#r1912969020 😊